### ISSUE TITLE
still missing accessibility report

### ISSUE THREAD

**Issue by J-D-CONNOR** [CSC207-UofT/course-project-not-a-team#142]
We are still missing accessibility report, also please try to format .md file in a good way so it is easy to read

**Comment by ZacYiYang** [comment]
We are working on that!

**Comment by J-D-CONNOR** [comment]
Finished it, and pull request approved by Zac.
